perceval/rendering/pdisplay.py

```diff
@@ -219,12 +226,89 @@ def pdisplay_state_distrib(sv: Union[StateVector, ProbabilityDistribution, BSCou
     s_states = tabulate(d, headers=headers, tablefmt=_TABULATE_FMT_MAPPING[output_format])
     return s_states
 
+    # labels on x- and y- axes
+
+
+def _generate_pauli_captions(nqubit):
+    from perceval.algorithm.tomography.tomography_utils import _generate_pauli_index
+    pauli_indices = _generate_pauli_index(nqubit)
+    pauli_names = []
+    for subset in pauli_indices:
+        pauli_names.append([member.name for member in subset])
+
+    basis = []
+    for val in pauli_names:
+        basis.append(''.join(val))
+    return basis
+
+
+def _get_sub_figure(ax, array, basis_name):
+    # Data
+    size = array.itemsize * 2
+    x = numpy.array([[i] * size for i in range(size)]).ravel()  # x coordinates of each bar
+    y = numpy.array([i for i in range(size)] * size)  # y coordinates of each bar
+    z = numpy.zeros(size * size)  # z coordinates of each bar
+    dxy = numpy.ones(size * size) * 0.5  # Width/Lenght of each bar
+    dz = array.ravel()  # length along z-axis of each bar (height)
+
+    # Colors
+    color_map = plt.cm.get_cmap('viridis_r')
```

Comment:
not going the colormap `cool`?

Reply:
No it's not color blind friendly :'(

Reply:
Too bad. Then let's keep the color-blind friendly one as default as you had.